spec/win32/unit/certstore_spec.rb

```diff
@@ -188,70 +188,6 @@
 
   CERT_SYSTEM_STORE_LOCAL_MACHINE = 0x00020000
 
-  describe "#get_cert_pfx" do
```

Comment:
We removed this, but did not add test for `#get_cert_key` which seems to replace it.